files/zh-tw/web/http/reference/status/205/index.md

```diff
@@ -2,27 +2,56 @@
 title: 205 Reset Content
 slug: Web/HTTP/Reference/Status/205
 l10n:
-  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
+  sourceCommit: 74ab26a101ef2e4d5e5f25962033bc1042102677
 ---
 
 {{HTTPSidebar}}
 
-HTTP **`205 Reset Content`** 成功回應碼告訴用戶端重置文件視圖，例如清除表單的內容、重置畫布狀態或刷新使用者介面。
+HTTP **`205 Reset Content`** [成功回應](/zh-TW/docs/Web/HTTP/Reference/Status#成功回應)狀態碼表示請求已成功處理，且用戶端應重置文件視圖。
+
+此回應旨在支援使用者接收支援資料輸入的內容、在請求中提交使用者編輯的資料，以及需要為下一次輸入重置內容的用例。「重置內容」的指令可能意味著清除表單內容、重置畫布狀態或刷新使用者介面；具體實作取決於用戶端。
+
+> [!NOTE]
+> 在使用 `205` 狀態的 Web 應用程式中，會假定用戶端在收到 `205` 回應後處理重置內容。這通常是透過 JavaScript 完成的，因為瀏覽器本身不會在收到 `205` 回應後原生處理重置表單等內容。
+
+請注意，回應不得包含任何內容或 {{HTTPHeader("Content-Length")}} 標頭（瀏覽器可能會拒絕包含內容的回應）。也可以使用帶有空區塊的 {{HTTPHeader("Transfer-Encoding", "Transfer-Encoding: chunked")}} 標頭來表示空回應。
```

Comment:
https://zh.wikipedia.org/zh-tw/%E5%88%86%E5%9D%97%E4%BC%A0%E8%BE%93%E7%BC%96%E7%A0%81

```suggestion
請注意，回應不得包含任何內容或 {{HTTPHeader("Content-Length")}} 標頭（瀏覽器可能會拒絕包含內容的回應）。也可以使用帶有空分塊的 {{HTTPHeader("Transfer-Encoding", "Transfer-Encoding: chunked")}} 標頭來表示空回應。
```